src/Bundle/spec/Grid/Renderer/TwigBulkActionGridRendererSpec.php

```diff
@@ -41,18 +45,15 @@ function it_is_a_bulk_action_grid_renderer(): void
     function it_uses_twig_to_render_the_bulk_action(
         Environment $twig,
         OptionsParserInterface $optionsParser,
-        ResourceGridView $gridView,
+        GridViewInterface $gridView,
         Action $bulkAction,
         RequestConfiguration $requestConfiguration,
         Request $request,
```

Comment:
Should it be removed as for `src/Bundle/spec/Grid/Renderer/TwigGridRendererSpec.php` (`use` as well)?
```suggestion
```